### ISSUE TITLE
Capture on VirtualBox VM doesn't work anymore

### ISSUE THREAD

**Issue by ehlers** [GNS3/gns3-server#105]
GNS3 version 1.3.0rc1, OS: Linux, Debian 8

Starting capture on a VirtualBox VM doesn't work anymore, on 1.3.0beta2 it was working.

GNS3 Console output:
```
Server error from 192.168.1.11:8000: ost-drone1: Traceback (most recent call last):
  File "/opt/GNS3/lib/python3.4/site-packages/gns3server/web/route.py", line 139, in control_schema
    yield from func(request, response)
  File "/usr/lib/python3.4/asyncio/coroutines.py", line 141, in coro
    res = func(*args, **kw)
  File "/opt/GNS3/lib/python3.4/site-packages/gns3server/handlers/api/virtualbox_handler.py", line 342, in start_capture
    vm.start_capture(adapter_number, pcap_file_path)
  File "/opt/GNS3/lib/python3.4/site-packages/gns3server/modules/virtualbox/virtualbox_vm.py", line 855, in start_capture
    if nio.capturing:
AttributeError: 'NoneType' object has no attribute 'capturing'
```

**Comment by grossmj** [comment]
@ehlers 

What is the network configuration for your VM? (how many adapters etc.)

Thanks,

**Comment by ehlers** [comment]
Here my VirtualBox configuration:
![vbox-configuration](https://cloud.githubusercontent.com/assets/896406/6790012/dc97ae80-d1a4-11e4-8a21-14516ca3a0ec.png)
![vbox-net1](https://cloud.githubusercontent.com/assets/896406/6790014/e39d99ba-d1a4-11e4-8af6-bc0fd87837fc.png)
![vbox-net1_forwarding](https://cloud.githubusercontent.com/assets/896406/6790016/e98845d2-d1a4-11e4-8b8f-6c276fab07e8.png)

And here my GNS3 project:
![gns3](https://cloud.githubusercontent.com/assets/896406/6790063/361476c8-d1a5-11e4-9c05-881d8159f188.png)
![gns3-drone1_net](https://cloud.githubusercontent.com/assets/896406/6790065/390aa6b8-d1a5-11e4-9222-6f3680cf8837.png)

And that's my ifconfig:
```
behlers@dell:~$ LANG=C /sbin/ifconfig
eth0      Link encap:Ethernet  HWaddr 5c:f9:dd:52:61:f4  
          UP BROADCAST MULTICAST  MTU:1500  Metric:1
          RX packets:0 errors:0 dropped:0 overruns:0 frame:0
          TX packets:0 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1000 
          RX bytes:0 (0.0 B)  TX bytes:0 (0.0 B)

lo        Link encap:Local Loopback  
          inet addr:127.0.0.1  Mask:255.0.0.0
          inet6 addr: ::1/128 Scope:Host
          UP LOOPBACK RUNNING  MTU:65536  Metric:1
          RX packets:507 errors:0 dropped:0 overruns:0 frame:0
          TX packets:507 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:0 
          RX bytes:41543 (40.5 KiB)  TX bytes:41543 (40.5 KiB)

tap0      Link encap:Ethernet  HWaddr 7a:9a:f3:e9:1e:d5  
          inet addr:172.31.1.254  Bcast:172.31.1.255  Mask:255.255.255.0
          UP BROADCAST MULTICAST  MTU:1500  Metric:1
          RX packets:0 errors:0 dropped:0 overruns:0 frame:0
          TX packets:0 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:500 
          RX bytes:0 (0.0 B)  TX bytes:0 (0.0 B)

wlan0     Link encap:Ethernet  HWaddr 84:a6:c8:e5:a3:da  
          inet addr:192.168.1.11  Bcast:192.168.1.255  Mask:255.255.255.0
          inet6 addr: fd50:ea59:d24e:0:86a6:c8ff:fee5:a3da/64 Scope:Global
          inet6 addr: fe80::86a6:c8ff:fee5:a3da/64 Scope:Link
          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
          RX packets:11 errors:0 dropped:0 overruns:0 frame:0
          TX packets:65 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1000 
          RX bytes:1305 (1.2 KiB)  TX bytes:11479 (11.2 KiB)

behlers@dell:~$
```

**Comment by ehlers** [comment]
I tried with a simpler configuration, a VirtualBox VM with one port. Then it works.
If I try with 4 ports on the VM and connect Ethernet0, it also works.
But if I leave Ethernet0 empty and connect Ethernet1., then I get this error.

It seems, that the empty Etthernet0 creates this problem.

**Comment by grossmj** [comment]
@ehlers thanks a lot, you have done all the debugging for me ;)

Unfortunately I could not fix it before we release RC2. However, I will definitely have it fixed soon, ready for 1.3 final.

Cheers,

**Comment by grossmj** [comment]
@ehlers 

This is fixed. The issue was on the client side. Let me know if you find other issues.

Thanks :)
